application-admintools-default/src/main/resources/templates/data/configurationTemplate.vm

```diff
@@ -0,0 +1,43 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+#if($configuration["serverFound"] != $null)
+|(% id="admin-tools-dash-content" %)(((
```

Comment:
You should use html for the complete template

Reply:
Use camelCase `adminToolsDashboardItem`
Also, why not use a class? If you have an id, that it cannot be used for the other sections (which should have the same display)

Reply:
A class is better use, but why use HTML for the template?

Reply:
> why use HTML for the template

Wiki syntax is not supported in a template by default (it's only Velocity producing directly the result sent to the browser). You can enable wiki content, but we often tend to use directly HTML as template are generally intended for UI stuff. See https://extensions.xwiki.org/xwiki/bin/view/Extension/Template%20Module#HContent for more details.